Add support for String delimiters in @CsvSource and @CsvFileSource (#1972)
junit-team/junit5: documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

include::{testDir}/example/ParameterizedTestDemo.java[tags=CsvSource_example]
----

The default delimiter is `,` but you can use another character by setting the `delimiter`

> attribute. The `delimiterString` attributes also allows you to use a String as a delimiter.

Reviewer: ```suggestion
attribute. Alternatively, the `delimiterString` attribute allows to use a `String` delimiter.
```
